src/common/config/config.production.json

```diff
@@ -1,13 +1,13 @@
 {
     "themeFile": "theme.pangea.css",
     "appLogoUrl": "/favicon.png",
-    "appName": "Pangea ID",
-    "ecosystemName": "Pangea",
+    "appName": "Tonomy ID",
+    "ecosystemName": "Tonomy",
     "appSlogan": "For a Better Tomorrow",
     "images": {
```

Comment:
would be better if the directory names are renamed. can you change this in future PR?